client/blocks/url-preview/index.jsx

```diff
@@ -17,10 +17,11 @@ import { getPreviewSite, getPreviewSiteId, getPreviewUrl } from 'state/ui/previe
 import { getSiteOption, getSiteSlug } from 'state/sites/selectors';
 import addQueryArgs from 'lib/route/add-query-args';
 import isDomainOnlySite from 'state/selectors/is-domain-only-site';
+import AsyncLoad from 'components/async-load';
 
 const debug = debugFactory( 'calypso:design-preview' );
 
-export default function urlPreview( WebPreview ) {
+export default function urlPreview() {
```

Comment:
After this change, `blocks/url-preview` is most likely broken. The module exports a function that returns a React class, but in `blocks/site-preview`, you import is as if it was a React class: `import UrlPreview from 'blocks/url-preview'`.

Note that when `WebPreview` is no longer a parameter and is hardcoded, this doesn't need to be a HOC any more -- just export a `UrlPreview` class.